dll/win32/shell32/shlfileop.cpp

```diff
@@ -153,7 +153,7 @@ static INT_PTR ConfirmMsgBox_Init(HWND hDlg, LPARAM lParam)
 
     SetWindowTextW(hDlg, info->lpszCaption);
     ShowWindow(GetDlgItem(hDlg, IDC_YESTOALL_MESSAGE), SW_HIDE);
-    SetPropW(hDlg, L"WINE_CONFIRM", info->lpszText);
+    SetWindowLongPtrW(hDlg, DWLP_USER, (LONG_PTR)info->lpszText);
```

Comment:
We were talking about the Prop leakage, but here ,is the `info->lpszText` pointer correctly dealt with (i.e. freed) when the dialog is terminated, if applicable?

Reply:
This is a modal dialog. The struct lives on the stack and the string itself is a parameter from the caller.